aspnetcore/migration/22-to-30.md

```diff
@@ -69,3 +69,22 @@ Newtonsoft settings can be set with `AddNewtonsoftJson`:
                options.SerializerSettings.ContractResolver = 
                   new CamelCasePropertyNamesContractResolver());
     ```
+
+## Generic host replaces WebHostBuilder
+
+The ASP.NET Core 3.0 templates use [Generic Host](fundamentals/host/generic-host). Previous versions used [WebHostBuilder](xref:fundamentals/host/web-host). The following code shows the ASP.NET Core 3.0 template generated `Program` class:
+
+[!code-csharp[Main](22-to-30/samples/Program.cs?name=snippet)]
+
+The following code shows the ASP.NET Core 2.2 template generated `Program` class:
+
+[!code-csharp[Main](22-to-30/samples/Program2.2.cs?name=snippet)]
+
+<xref:Microsoft.AspNetCore.Hosting.IWebHostBuilder> is used with the <xref:Microsoft.AspNetCore.Hosting.WebHostBuilder> (in the 2.2 templates) and with `webBuilder` (in the 3.0 templates). `WebHostBuilder` will be deprecated in a future release, and eventually removed.  `HostBuilder` will replace `WebHostBuilder`.
```

Comment:
Used with? ... `WebHostBuilder` implements `IWebHostBuilder` to build an `IWebHost`.

I can't review all of the first sentence ... I need to dig into it more because the sand has perhaps shifted a bit since 2.2 (it was/is `HostBuilder` implements `IHostBuilder` to build an `IHost`). I've been 🏃 on other issues. If you want to wait until Thursday, I can take a closer look and then review these remarks.

Last two sentences ok, but there's a comma splice (before "and") and an extra space after "removed."